core/core.go

```diff
@@ -72,7 +72,7 @@ func Start() error {
 	// start the rtmp server
 	go rtmp.Start(setStreamAsConnected, setBroadcaster)
 
-	port := config.WebServerPort
+	port := data.GetHTTPPortNumber()
```

Comment:
This isn't correct if somebody overrides the port on the command line.  This is necessary if somebody is already running something on port 8080 and can't get to the admin to change it.

It should be getting set in https://github.com/owncast/owncast/blob/master/main.go#L112

Reply:
Heh, you're right but there's a race condition which is causing the logging to be incorrect.

https://github.com/owncast/owncast/blob/1d9774e4533512ce708bdd7abeb10fa7e6a93806/main.go#L107-L109
Is called **before** the `config.WebServerPort` is set:
https://github.com/owncast/owncast/blob/1d9774e4533512ce708bdd7abeb10fa7e6a93806/main.go#L111-L122

Reply:
ah yeah good catch!  `core.Start()` shouldn't be logging the web server port, that should be in `router.Start()`.

Reply:
Logging moved to the appropriate package. :)